src/Synapse/Synapse/help/Remove_AzSynapseSqlPoolRestorePoint.md

```diff
@@ -0,0 +1,291 @@
+﻿---
```

Comment:
Just curious, why the file name is Remove_AzSynapseSqlPoolRestorePoint rather than Remove-AzSynapseSqlPoolRestorePoint ?

Reply:
typo error, let me correct